tests/docker/globus.el8.Dockerfile

```diff
@@ -0,0 +1,88 @@
+FROM almalinux:8
+
+RUN \
+  dnf update -y && \
+  dnf install -y \
+    pam-devel \
+    python3-jsonschema \
+    epel-release \
+    gcc-c++ \
+    gnupg \
+    make \
+    python3 \
+    python3-pip \
+    rsyslog \
+    sudo \
+    wget \
+    which \
+    diffutils \
+    procps \
+    rpm-build \
+    authd \
+    unixODBC-devel \
+  && \
+  dnf clean all && \
+  rm -rf /var/cache/dnf /tmp/*
+
+# python 2 and 3 must be installed separately because dnf will ignore/discard python2
+RUN \
+  dnf check-update -q >/dev/null || { [ "$?" -eq 100 ] && dnf update -y; } && \
```

Comment:
I'd recommend against doing this. If you need to update all eligible packages in a docker image, do it in a separate `RUN` block before any `RUN` blocks that use `dnf`. The process may be repeated if new repositories are added. This applies to all usage of this pattern and `dnf update` in general in Dockerfiles.

Reply:
This was copied from the development environment project files when told to do it like in the development environment.  Then we changed yum to dnf.

We'll have to discuss this.

Reply:
I made this change to put this first thing in its own RUN block for both el8 and el9.  Tests passed.